src/apps/routes/tools.clj

```diff
@@ -16,11 +17,13 @@
                            user-get-tool]]
         [apps.tools.private :only [add-private-tool delete-private-tool update-private-tool]]
         [apps.user :only [current-user]]
+        [apps.util.coercions :only [coerce!]]
         [apps.util.service]
         [slingshot.slingshot :only [throw+]]
         [ring.util.http-response :only [ok]])
   (:require [apps.routes.schemas.permission :as permission]
             [apps.service.apps :as apps]
+            [apps.service.apps.de.listings :as app-listings]
```

Comment:
This would have to be done as part of a larger refactor, but I'd eventually like to have all app listing operations go directly through `apps.service.apps`.